src/main/java/uk/gov/hmcts/reform/ccd/test/stubs/service/controllers/StubResponseController.java

```diff
@@ -43,6 +56,7 @@
 public class StubResponseController {
 
     private static final Logger LOG = LoggerFactory.getLogger(StubResponseController.class);
+    static final String WIREMOCK_STUB_MAPPINGS_ENDPOINT = "/__admin/mppings";
```

Comment:
```suggestion
    static final String WIREMOCK_STUB_MAPPINGS_ENDPOINT = "/__admin/mappings";
```

Reply:
Ah! I did change that last minute to simulate an error and forgot to revert it :) 
Thanks @danlysiak